node_modules/gh-apps/lib/rest.js

```diff
@@ -163,3 +163,26 @@ var _updateAppAdmins = function(req, res) {
 
 GrassHopper.globalAdminRouter.on('post', '/api/apps/:id/admins', _updateAppAdmins);
 GrassHopper.appRouter.on('post', '/api/apps/:id/admins', _updateAppAdmins);
+
+/**
+ * @REST getTermsAndConditions
+ *
+ * Get the Terms and Conditions
+ *
+ * @Server      admin,app
+ * @Method      GET
+ * @Path        /apps/{id}/termsAndConditions
+ * @PathParam   {number}            id              The id of the app for which to get the Terms and Conditions
+ * @Return      {TermsAndConditions}                The Terms and Conditions for the app
+ */
+var _getTermsAndConditions = function(req, res) {
+    AppsAPI.getTermsAndConditions(req.ctx, req.params.id, function(err, termsAndConditions) {
+        if (err) {
+            return res.status(err.code).send(err.msg);
+        }
+
+        return res.status(200).send(termsAndConditions);
+    });
+};
+GrassHopper.globalAdminRouter.on('get', '/api/apps/:id/termsAndConditions', _getTermsAndConditions);
```

Comment:
Needs new line before this. (reposing because other was a commit comment and not PR)